src/Resources/contao/dca/tl_theme.php

```diff
@@ -230,260 +230,293 @@
 /**
  * Provide miscellaneous methods that are used by the data configuration array.
  *
- * @author Leo Feyer <https://github.com/leofeyer>
+ * @author     Leo Feyer <https://github.com/leofeyer>
+ *
+ * @deprecated Deprecated since Contao 4.6, to be removed in Contao 5.0
+ *             Use the datacontainer.theme listener instead.
```

Comment:
Not sure if we should write this. To me, an event listener is not public API. I never rely on a listener to be reusable for me. No idea why one would rely on this to be honest.

Reply:
Consider the following example: https://github.com/inspiredminds/contao-sibling-navigation/blob/1.0.1/src/Resources/contao/dca/tl_module.php#L41-L49
This extension overrides the `options_callback` of the `news_order` field of `tl_module`, returns custom options based on certain conditions and otherwise executes the original callback function.

If that is not possible then you would always have to copy the original code into your callback, which seems unnecessary.